presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -44,11 +44,13 @@ Date and Time Functions
 
 .. data:: current_time
 
-    Returns the current time with time zone as of the start of the query.
+    Returns the current time with session time zone as of the start of the
+    query.
```

Comment:
Wrapping a single word decreases readability. We don't have a hard limit, so no need.

Reply:
Actually, let's revert these changes for now. It reads strangely, since "as of the start" sounds like it binds/applies to the "session time zone" part. I'm actually more concerned about fixing the "current time zone" language for the `current_timezone()` function, but we can follow up on that.

Reply:
Done